cmds/ocm/commands/ocmcmds/common/inputs/types/dockermulti/type.go

```diff
@@ -14,8 +14,8 @@ func init() {
 const usage = `
 This input type describes the composition of a multi-platform OCI image.
 The various variants are taken from the local docker daemon. They should be 
-built with the buildx command for cross platform docker builds.
-The denoted images, as well as the wrapping image index is packed as OCI
+[built with the `buildx` command](https://github.com/open-component-model/ocm-website/blob/main/content/docs/tutorials/best-practices-with-ocm.md#building-multi-arch-images) for cross platform docker builds.
```

Comment:
@frewilhelm good catch. The docs under https://github.com/open-component-model/ocm/tree/main/docs/reference are generated from this source and what I pasted into the code is just the desired result. What would be the correct way to do it? Are there anything similar like escape characters that can be used in this case?

Reply:
@mandelsoft @fabianburth do you have an idea?

Reply:
I discussed with @mandelsoft it is not possible to embed links as in markdown. However, the link is referring to a markdown file in the GitHub repository. But the content is also available on the website. Thus, I would suggest to link to the website. This will shorten the length of the link a bit^^

```suggestion
built with the "buildx" command for cross platform docker builds (see https://ocm.software/docs/tutorials/best-practices/#building-multi-architecture-images) .
```

Reply:
👍🏻